apps/playoff-schedule/backend/src/main.ts

```diff
@@ -0,0 +1,65 @@
+// בס"ד
+import express from "express";
+import { apiRouter } from "./routes";
+import cors from "cors"; 
+
+const app = express();
+app.use(cors({
+    origin: "*"
+}));
+
+app.use("/api/v1", apiRouter);
+
+const defaultPort = 4590;
+const port = process.env.BACKEND_PORT ?? defaultPort;
+
+app.use(express.json({ limit: "10mb" }));
+app.use(express.urlencoded({ limit: "10mb", extended: true }));
+const statusBadServer = 500;
+const statusGood = 200;
+const statusBadUser = 400;
+
+// enter a valid api key
+const apiKey = "YOUR_API_KEY";
```

Comment:
consider using enviromental variables to pass in sensitive information. create a `.secret.env` file near the `.public.env` file (the `.secret.env` file is already gitignored and used in the files). consider looking at the `.public.env` at how to make enviromental variables. 
```suggestion
const apiKey = process.env.TBA_API_KEY ?? "YOUR_API_KEY";
```